tests/common-unit/hierarchical-menu/index.ts

```diff
@@ -0,0 +1,10 @@
+import type { TestSetup } from '../common';
+import { createOptimisticUiTests } from './optimistic-ui';
+
+export type HierarchicalMenuSetup = TestSetup<{ attributes: string[] }>;
```

Comment:
Could it be worth allowing to pass all widget parameters right now? If later we need them, the diff will be smaller for existing tests that implement the common test suite.

Same for the other tests.

Reply:
Ah yes, I forgot that we have the type for the widgetParams. Totally makes sense to add